docs/main/user-guides/local-engine-user-guide/local-pv-lvm/lvm-deployment.md

```diff
@@ -0,0 +1,235 @@
+ DataSource	          -	                                Not Supported	        Pending
+
+ ## PersistentVolumeClaim Parameters
+
+ **AccessMode**
+
+LVM-LocalPV supports only ReadWriteOnce access mode i.e volume can be mounted as read-write by a single node. AccessMode is a required field, if the field is unspecified then it will lead to a creation error. For more information about access modes workflow click here.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: csi-lvmpv
+spec:
+  accessModes:
+    - ReadWriteOnce        ## Specify ReadWriteOnce(RWO) access modes
+  storageClassName: openebs-lvm
+  resources:
+    requests:
+      storage: 4Gi
+```
+
+**StorageClassName**
+
+LVM CSI-Driver supports dynamic provision of volume for the PVCs referred to lvm storageclass. StorageClassName is a required field, if field is unspecified then it will lead to a provision errors. For more information about dynamic provisioning workflow click here.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: csi-lvmpv
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: openebs-lvm    ## It must be OpenEBS LVM storageclass for provisioning LVM volumes
+  resources:
+    requests:
+      storage: 4Gi
+```
+
+**Capacity Resource**
+
+Admin/User can specify the desired capacity for lvm volume. CSI-Driver will provision a volume if the underlying volume group has requested capacity available else provisioning volume will be errored. StorageClassName is a required field, if field is unspecified then it will lead to a provision errors. For more information about workflow click here
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: csi-lvmpv
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: openebs-lvm
+  resources:
+    requests:
+      storage: 4Gi       ## Specify required storage for an application
+```
+
+**VolumeMode (Optional)**
+
+LVM-LocalPV supports two kind of volume modes(Defaults to Filesystem mode):
+
+Block (Block mode can be used in a case where application itself maintains filesystem)
+Filesystem (Application which requires filesystem as a prerequisite) Note: If unspecified defaults to Filesystem mode. More information about workflow is available here.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: csi-lvmpv
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: openebs-lvm
+  volumeMode: Filesystem     ## Specifies in which mode volume should be attached to pod
+  resources:
+    requests:
+      storage: 4Gi
+```
+
+**Selectors (Optional)**
+
+Users can bind any of retained lvm volumes to new PersistentVolumeClaim object via selector field. If selector and volumeName fields are unspecified then LVM CSI driver will provision new volume. If volume selector is specified then request will not reach to localpv driver this is a use case of pre-provisioned volume, for more information about workflow click here
+
```

Comment:
Change this to "See here for more information about workflow"

(Applicable for all similar occurrences)